File: problems/0034 - Find First ans Last Position of Element in Sorted Array/Go/Solution.go
func searchRange(nums []int, target int) []int {
	first, last := -1, -1

	for i, v := range nums {
		if v == target {
			if first == -1 {
				first = i
			}

			last = i
		}
	}

	return []int{first, last}
}

File: problems/0034 - Find First ans Last Position of Element in Sorted Array/PHP/Solution.php
<?php

class Solution {

/**
 * @param Integer[] $nums
 * @param Integer $target
 * @return Integer[]
 */
function searchRange($nums, $target) {
    $first = -1;
    $last = -1;

    foreach ($nums as $key => $value) {
        if ($value === $target) {
            if ($first === -1) {
                $first = $key;
            }

            $last = $key;
        }
    }

    return [$first, $last];
}
}